doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -804,6 +804,7 @@ Other samples
 * :ref:`radio_test` sample:
 
   * Updated the sample by aligning the timer's configuration to the new nrfx API.
+  * Corrected setting the TX power with FEM.
```

Comment:
```suggestion
  * Corrected the way of setting the TX power with FEM.
```
BTW, why in release notes 2.5.0 and not in the changelog?

Reply:
This was a mistake when selecting file from list.